src/index.js

```diff
@@ -1,35 +1,55 @@
-// import _ from 'lodash';
-
 import './style.css';
+import activeCheckbox from './check.js';
+
+window.onload = function () {
+  const todoList = document.querySelector('.todo-list');
+
+  const listTable = [
+    {
+      description: 'go to the market',
+      completed: false,
+      index: 0,
+    },
+    {
+      description: 'wash dishes',
+      completed: false,
+      index: 1,
+    },
+    {
+      description: 'Complete my project',
+      completed: false,
+      index: 2,
+    },
+    {
+      description: 'do sport',
+      completed: false,
+      index: 3,
+    },
+  ];
+
+  let dayTodo = [];
+  const localStoregeContent = localStorage.getItem('todo');
+  if (localStoregeContent === null) {
+    dayTodo = listTable;
+  } else {
+    dayTodo = JSON.parse(localStoregeContent);
+  }
 
-const listTable = [
-  {
-    description: 'go to the market',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'wash dishes',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'play guitar',
-    completed: true,
-    index: 2,
-  },
-  'rwanda',
-];
+  localStorage.setItem('todo', JSON.stringify(dayTodo));
+  console.log(dayTodo);
+  for (let i = 0; i < listTable.length; i += 1) {
+    listTable[i] = dayTodo[i];
+  }
 
-const todoList = document.querySelector('.todo-list');
-// eslint-disable-no-use-before-define
-function displayTodo(item) {
-  return ` <div class='todo'>
+  function displayTodo(item) {
```

Comment:
- [ ] Part of ES6 good practices is to always use arrow functions, it looks like the code reviewers before told you that, and it looks like this function is not an arrow function yet, please fix it. 🐶